controller/setting_controller.go

```diff
@@ -1084,6 +1084,16 @@ func (sc *SettingController) syncDefaultLonghornStaticStorageClass() error {
 	}
 
 	defaultStaticStorageClassName := setting.Value
+
+	definition, ok := types.GetSettingDefinition(types.SettingNameDefaultLonghornStaticStorageClass)
+	if !ok {
+		return fmt.Errorf("setting %v is not supported", types.SettingNameDefaultLonghornStaticStorageClass)
+	}
+
+	if defaultStaticStorageClassName != definition.Default {
+		return nil
+	}
+
```

Comment:
Could you leave a comment for the purpose of the check? The comment will help developers understand the context. Thanks.

Reply:
Added. Thanks.